Changed Player Movement and updated textures

joelbeedle opened the pull request.

joelbeedle committed:
Changed Player Movement and updated textures

wtljohnson committed:
Merge branch 'main' into Refactor-Player-Movement

# Conflicts:
#	core/src/com/boatcorp/boatgame/screens/PlayScreen.java

wtljohnson reviewed (changes requested):
Please reimplement the points system

joelbeedle committed:
Refactor PointSystem and Player

Fixed points not being applied to movement

Added speed & point scaling so holding down two buttons doesn't affect the speed or point gain.

joelbeedle committed:
Fix code implementation issues

Changed code to be nicer

wtljohnson reviewed (approved):
Looks good
